README.md

```diff
@@ -7,6 +7,10 @@
 
 Fabric React is a responsive, mobile-first collection of robust components designed to make it quick and simple for you to create web experiences using the Office Design Language.
 
+# [Wiki](https://github.com/OfficeDev/office-ui-fabric-react/wiki)
+
+### https://github.com/OfficeDev/office-ui-fabric-react/wiki

```

Comment:
>### [](start = 0, length = 3)

Rather than use a section header for the link with no other context, a suggestion is to see text instead introducing the wiki, with a link there.